internal/plugins/helm/v1/scaffolds/templates/manager/config.go

```diff
@@ -87,9 +87,7 @@ spec:
         control-plane: controller-manager
     spec:
       containers:
-      - args:
-        - manager
```

Comment:
This removal is no directly related to this PR, but needs to happen to avoid problems running the operator in a cluster. 

This is an accidental carryover from Go operators that have a `manager` binary. But helm-operator has a different binary entrypoint: the base image adds the binary and sets the entrypoint for us, so we just need to specify the flags to pass to the helm-operator binary.